lib/ansible/modules/cloud/azure/azure_rm_webapp.py

```diff
@@ -0,0 +1,930 @@
+            - Only applys for windows web app.
+
+    linux_fx_version:
+        description:
+            - The runtime stack used for your linux-based webapp, e.g., "RUBY|2.3", "NODE|6.6", "PHP|5.6", "DOTNETCORE|1.1.0". 
+            - Only applys for linx web app. See https://aka.ms/linux-stacks for more info.
+
+    java_container_settings:
+        description: Java container settings.
+        suboptions:
+            name:
+                description: The java container, e.g., Tomcat, Jetty.
+            value:
+                description: The version of the java container, e.g., '8.0.23' for Tomcat.
+
+    container_settings:
+        description: Web app container settings.
+        suboptions:
+            name:
+                description: Name of container. eg. imagename:tag
+            registry_server_url:
+                description: Container registry server url. eg. mydockerregistry.io
+            registry_server_user:
+                description: The container registry server user name.
+            registry_server_password:
+                description:
+                    - The container registry server password.
+    
+    scm_type:
+        description:
+            - Repository type of deployment source. Eg. LocalGit, GitHub.
+    
+    deployment_source:
+        description:
+            - Deployment source for git
+
+            suboptions:
+                url:
+                    description:
+                        - Repository url of deployment source.
+                
+                branch:
+                    description:
+                        - The branch name of the repository.
+    startup_file:
+        description:
+            - The web's startup file.
+            - This only applys for linux web app.
+        
+
+    client_affinity_enabled:
+        description:
+            - True to enable client affinity; False to stop sending session affinity cookies, which route client requests in the
+            - same session to the same instance.
+        default: True
+
+    https_only:
+        description:
+            - Configures web site to accept only https requests.
+
+    skip_dns_registration:
+        description:
+            - If true web app hostname is not registered with DNS on creation. This parameter is
+            - only used for app creation.
+
+    skip_custom_domain_verification:
+        description:
+            - If true, custom (non *.azurewebsites.net) domains associated with web app are not verified.
+
+    force_dns_registration:
+        description:
+            - If true, web app hostname is force registered with DNS.
+
+    ttl_in_seconds:
+        description:
+            - Time to live in seconds for web app's default domain name.
+    
+    app_settings:
+        description:
+            - Configure web app application settings. Suboptions are in format: <yourKey>: <yourValue>.
```

Comment:
`format C(<yourKey>: <yourValue>)`
the ` : ` is confusing the yaml parser
Using `C(..)` will give you inline code formattign

Reply:
thank you!